core/src/main/java/hudson/model/TimeZoneProperty.java

```diff
@@ -0,0 +1,118 @@
+package hudson.model;
+
+import hudson.Extension;
+import hudson.util.FormValidation;
+import hudson.util.ListBoxModel;
+import hudson.util.ListBoxModel.Option;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.TimeZone;
+import java.util.logging.Logger;
+import java.util.logging.Level;
+
+/**
+ * A UserProperty that allows a user to specify a time zone for displaying time.
+ *
```

Comment:
`@since TODO` or `@Restricted`

Reply:
Could you clarify? Do you just wanted me to add @Restricted to the class?

Reply:
Either add `@since` to indicate when this was added to the public API of Jenkins, or `@Restricted` to prevent use in plugins. No strong opinion which.